third_party/xla_client/pjrt_computation_client.cc

```diff
@@ -528,6 +527,10 @@ PjRtComputationClient::ExecuteReplicated(
   execute_options.strict_shape_checking = true;
   // TODO(yeounoh) currently only support single-slice execution
   execute_options.multi_slice_config = nullptr;
+
+  // Required as of cl/518733871
```

Comment:
do not include cl number in open pr, use pr number on public github instead.(fix in next pr).